app/helpers/games_helper.rb

```diff
@@ -18,9 +18,9 @@ def get_scoreboard_text(registration)
     when Registration::ZOMBIE_FACTION
       dies_in = (registration.time_until_death / 1.hour).ceil
       tag_string = "#{scoreboard_highlight(tag_count)} #{tag_count == 1 ? 'tag' : 'tags'}"
-      starve_string = "(Starves in less than #{scoreboard_highlight(dies_in)} hours)"
+      
 
-      [tag_string, starve_string].join(' ').html_safe
+      [tag_string].join(' ').html_safe
```

Comment:
`.join(' ')` is just going to join all the strings in the array with a space, so if theres only one string, we might as well just return that string. So we can change this to `tag_string.html_safe`